### ISSUE TITLE
Register signals to close controller

### ISSUE THREAD

**Issue by Skorlok** [lythx/trakman#224]
I use Trakman on Docker and it's boring to wait 10 seconds for the controller to get killed

```js
process.on("SIGINT", () => { this.end(); })
process.on("SIGTERM", () => { this.end(); })
process.on("SIGHUP", () => { this.end(); })
process.on("SIGBREAK", () => { this.end(); })
```
```js
process.exit(0)
```
this is how we do in JS

**Comment by wsrvn** [comment]
> boring to wait 10 seconds

watch some tiktok in the meantime

**Comment by felacek** [comment]
Fixed in #257
